core/templates/components/question-directives/question-misconception-selector/question-misconception-selector.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Oppia Authors. All Rights Reserved.
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
It is not necessary to change this unless you're creating a new file.

Reply:
Done.